.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Release .deb files'
+on: release
+
+jobs:
+  release-matrix:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        base_image: ["ubuntu:bionic", "ubuntu:disco", "ubuntu:eoan", "debian:buster"]
+    steps:
+      - uses: actions/checkout@v1
+      - name: docker build with arg
+        run: |
+          mkdir -p output
+          image_name=$(echo ${{matrix.base_image}} | cut -d: -f2 )
+          docker build --build-arg base_image=${{ matrix.base_image }} -t ${image_name} -f packaging/deb.dockerfile .
+          docker run --rm --mount "type=bind,source=$(pwd)/output,target=/nginx-sxg-module/output" ${image_name}
+          dpkg -I output/*.deb
+      - name: Upload to release
+        uses: JasonEtco/upload-to-release@master
```

Comment:
never depends on third party's master branch.

Reply:
switched to `kumagi/upload-to-release@v0.1.2`